obspy/io/stationxml/tests/test_stationxml.py

```diff
@@ -688,6 +692,113 @@ def test_numbers_are_written_to_poles_and_zeros(self):
             '<Pole number="1"> <Real>2.0</Real> '
             '<Imaginary>3.0</Imaginary> </Pole>', data)
 
+    def test_stationxml_with_custom_tags(self):
+        """
+        Test reading/writing StationXML with additional custom namespace tags.
+        """
+        filename = os.path.join(
+            self.data_dir, "IRIS_single_channel_with_response_custom_tags.xml")
+        # don't explicitely specify format
+        inv = obspy.read_inventory(filename)
+        self.assertTrue(hasattr(inv, "extra"))
+        self.assertTrue(hasattr(inv[0], "extra"))
+        self.assertTrue(hasattr(inv[0][0], "extra"))
+        self.assertTrue(hasattr(inv[0][0][0], "extra"))
```

Comment:
Please also test the actual tags and their values. It might also be useful to use slightly differnet tags + values in the test file to make sure the expected thing is returned.